internal/output/column.go

```diff
@@ -1,25 +1,10 @@
 package output
 
 import (
-	"encoding/json"
-	"fmt"
-	"math/big"
-	"reflect"
-	"strings"
-	"time"
-
-	"github.com/hazelcast/hazelcast-go-client/serialization"
-	"github.com/hazelcast/hazelcast-go-client/types"
-
-	"github.com/hazelcast/hazelcast-commandline-client/errors"
 	iserialization "github.com/hazelcast/hazelcast-commandline-client/internal/serialization"
 )
 
-type Column struct {
-	Name  string
-	Type  int32
-	Value any
-}
+type Column = iserialization.Column
```

Comment:
Why do we implement special methods for `string`, `nil` and `skip`? Can't we use NewValueColumn since already we are using it for most of the types?

Reply:
Removed helpers at: https://github.com/hazelcast/hazelcast-commandline-client/pull/203/commits/d1951c4d77fe39388b506e7553f5776a5fc09271